.github/workflows/build.sh

```diff
@@ -1,6 +1,6 @@
 
 if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-  if [[ "$MB_PYTHON_VERSION" == "pypy3.6-7.3" ]]; then
+  if [[ "$MB_PYTHON_VERSION" == pypy3* ]]; then
```

Comment:
Testing, I find this block is no longer necessary. I've created https://github.com/mattip/pillow-wheels/pull/5 to remove it.